lib/manager/helmv3/artifacts.ts

```diff
@@ -0,0 +1,76 @@
+import { quote } from 'shlex';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import {
+  getSiblingFileName,
+  getSubDirectory,
+  readLocalFile,
+  writeLocalFile,
+} from '../../util/fs';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+
+async function helmUpdate(manifestPath: string): Promise<void> {
+  const cmd = `helm dependency update ${quote(getSubDirectory(manifestPath))}`;
+
+  const execOptions: ExecOptions = {
+    docker: {
+      image: 'renovate/helmv3',
```

Comment:
I chose to called this `helmv3` not `helm`, in case `helm` is needed later for the v2 manager

Reply:
we can maybe to it using tags though, instead of putting it in the image/repository name

Reply:
We could - don't know if that would make the source of that repo messy? It would need to publish two different images. I don't mind how this is done anyway :)

Reply:
@mikebryant Don't worry about the image creation, I'll do it pretty easy. 🙃 I also don't think we need helm v2 images, as helm v2 become deprecated since helm v3 and will be unsupported in near future.

Reply:
Thanks. I'll update this PR to be `renovate/helm` then